test/parallel/test-eslint-lowercase-name-for-primitive.js

```diff
@@ -22,20 +22,28 @@ new RuleTester().run('lowercase-name-for-primitive', rule, {
   ],
   invalid: [
     {
-      code: 'new errors.TypeError("ERR_INVALID_ARG_TYPE", "a", "Number")',
-      errors: [{ message: 'primitive should use lowercase: Number' }]
+      code: `new errors.TypeError('ERR_INVALID_ARG_TYPE', 'a',
+            'Number')`,
```

Comment:
@shobhitchittora Could you change these backquote to singlequote? For example:

```js
{
  code: 'new errors.TypeError(\'ERR_INVALID_ARG_TYPE\', \'a\', ' +
        '\'Number\''
}
```

In addition, you can use this command to check if your change will pass CI:
```
node test/parallel/test-eslint-lowercase-name-for-primitive.js 
```

Reply:
Done and started a build.